src/cpl/mct/lnd_import_export.F90

```diff
@@ -189,23 +194,26 @@ real(r8) :: tdc, t               ! Kelvins to Celcius function and its input
        wateratm2lndbulk_inst%forc_rain_not_downscaled_grc(g)  = forc_rainc + forc_rainl
        wateratm2lndbulk_inst%forc_snow_not_downscaled_grc(g)  = forc_snowc + forc_snowl
 
+! TODO: Delete these lines after testing, except the call QSat
        if (forc_t > SHR_CONST_TKFRZ) then
           e = esatw(tdc(forc_t))
        else
           e = esati(tdc(forc_t))
        end if
-       qsat           = 0.622_r8*e / (forc_pbot - 0.378_r8*e)
+       qsat_old       = 0.622_r8*e / (forc_pbot - 0.378_r8*e)
+       call QSat_temp(forc_t, forc_pbot, dum1, dum2, qsat_kg_kg, dum3)
+       if (qsat_kg_kg - qsat_old > 1.e-14_r8) write(iulog,*) 'qsat_new, qsat_old =', qsat_kg_kg, qsat_old  ! slevis diag
```

Comment:
In the subr. call above, QSat_temp returns round-off diffs, while QSat returns diffs that exceed round-off.

I propose that we call QSat for consistency throughout the CTSM, despite the larger diffs.

Reply:
@billsacks I will request a code review of my changes this far, in order to initiate a discussion on whether this solution is acceptable.

Once this piece is resolved, we can discuss the functionalizing of remaining sections of repeating code in the caps.